.github/workflows/svm-exampls.yml

```diff
@@ -49,4 +49,4 @@ jobs:
       - name: Run tests
         run: |
           cd svm/examples
-          cargo test
\ No newline at end of file
+          cargo nextest run
```

Comment:
could we create a separate PR for it? btw, the pipeline isn’t triggered because the file path doesn’t include the pipeline itself. that needs to be updated as well